src/ui/components/NetworkMonitor/utils.ts

```diff
@@ -49,6 +53,18 @@ const name = (url: string): string =>
     .split("/")
     .filter(f => f.length)
     .pop() || "";
+const queryParams = (url: string): [string, string][] => {
+  //@ts-ignore
+  return Array.from(new URL(url).searchParams.entries() as [string, string][]);
+};
+const documentType = (headers: Header[]): string => {
+  return (
+    headers
+      .find(h => h.name.toLowerCase() === "content-type")
+      ?.value.split(",")[0]
+      .split(";")[0] || "unknown"
+  );
+};
```

Comment:
had to squint here, but looks legit.

Reply:
Yeah, this is a little inscrutable. Let me write a slightly clearer version.